libraries/AP_HAL_Linux/RPIOUARTDriver.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <AP_HAL/utility/RingBuffer.h>
```

Comment:
Not really. The parent class now has a class of that header so it is the responsibility of the parent class header to include the proper header. I'll defer to @lucasdemarchi to decide on this.